examples/tpm_io.c

```diff
@@ -62,15 +62,25 @@
     #else
         /* SPI */
         #ifdef WOLFTPM_MCHP
-            /* Microchip ATTPM20 */
-            /* SPI uses CE0 */
-            #define TPM2_SPI_DEV "/dev/spidev0.0"
+            /* Microchip ATTPM20 uses CE0 */
+            #define TPM2_SPI_DEV_CS "0"
         #elif defined(WOLFTPM_ST33)
             /* STM ST33HTPH SPI uses CE0 */
-            #define TPM2_SPI_DEV "/dev/spidev0.0"
+            #define TPM2_SPI_DEV_CS "0"
+        #elif defined(WOLFTPM_NUVOTON)
+            /* Nuvoton NPCT75x uses CE0 */
+            #define TPM2_SPI_DEV_CS "0"
         #else
             /* OPTIGA SLB9670 and LetsTrust TPM use CE1 */
-            #define TPM2_SPI_DEV "/dev/spidev0.1"
+            #define TPM2_SPI_DEV_CS "1"
+        #endif
+
+        #ifdef WOLFTPM_AUTODETECT
+            #undef TPM2_SPI_DEV
+            static char TPM2_SPI_DEV[] = "/dev/spidev0.0";
```

Comment:
Adding a short comment here that the autodetect changes the CS might be nice. Because initially I was confused why the autodetec has a fixed/hardcoded value, then saw the string manipulation later ;-)